Modules/Course/classes/class.ilObjCourseGUI.php

```diff
@@ -946,77 +946,146 @@ public function updateObject()
 				$this->object->setStatusDetermination((int) $form->getInput('status_dt'));
 			}
 		}
-		
+
 		if(!$old_autofill && $this->object->hasWaitingListAutoFill())
 		{
 			$this->object->handleAutoFill();
 		}
 		$this->object->update();
-		
-		
+
+
 		include_once './Services/Object/classes/class.ilObjectServiceSettingsGUI.php';
 		ilObjectServiceSettingsGUI::updateServiceSettingsForm(
 			$this->object->getId(),
 			$form,
 			array(
 				ilObjectServiceSettingsGUI::CALENDAR_VISIBILITY,
 				ilObjectServiceSettingsGUI::USE_NEWS,
-				ilObjectServiceSettingsGUI::AUTO_RATING_NEW_OBJECTS,				
+				ilObjectServiceSettingsGUI::AUTO_RATING_NEW_OBJECTS,
 				ilObjectServiceSettingsGUI::TAG_CLOUD,
 				ilObjectServiceSettingsGUI::CUSTOM_METADATA,
 				ilObjectServiceSettingsGUI::BADGES
 			)
 		);
-		
+
 		require_once('Services/Tracking/classes/class.ilChangeEvent.php');
 		global $ilUser;
 		ilChangeEvent::_recordWriteEvent($this->object->getId(), $ilUser->getId(), 'update');
-		ilChangeEvent::_catchupWriteEvents($this->object->getId(), $ilUser->getId());			
+		ilChangeEvent::_catchupWriteEvents($this->object->getId(), $ilUser->getId());
 
 		// lp sync confirmation required
 		if($show_lp_sync_confirmation)
 		{
 			return $this->confirmLPSync();
 		}
-		
+
 		// Update ecs export settings
-		include_once 'Modules/Course/classes/class.ilECSCourseSettings.php';	
-		$ecs = new ilECSCourseSettings($this->object);			
+		include_once 'Modules/Course/classes/class.ilECSCourseSettings.php';
+		$ecs = new ilECSCourseSettings($this->object);
 		if(!$ecs->handleSettingsUpdate())
 		{
 			$form->setValuesByPost();
 			ilUtil::sendFailure($GLOBALS['DIC']->language()->txt('err_check_input'));
 			return $this->editObject($form);
-		}			
+		}
+
+		// cat-tms-patch start
+
+		// venues (plugin)
+		if(ilPluginAdmin::isPluginActive('venues')) {
+
+			$vplug_data = ilPluginAdmin::getAllPlugins()['venues'];
+			$vplug = ilPluginAdmin::getPluginObject(
+				$vplug_data['component_type'],
+				$vplug_data['component_name'],
+				$vplug_data['slot_id'],
+				$vplug_data['name']
+			);
```

Comment:
We are Plugin-Maintainer now. Please create a PR for the `ilPluginAdmin::getActivePlugin($name)` sequence that is encoded here and that we already talked about. Create a JIRA ticket for the task, that also contains a note to cherry-pick the changes into this branch and adjust this location once the PR is merged.